app/containers/artist/components/header.js

```diff
@@ -0,0 +1,101 @@
+"use strict"
```

Comment:
more or less a direct port of the [one from Emission](https://github.com/artsy/emission/blob/master/lib/containers/artist.js)